bigflow/__init__.py

```diff
@@ -11,6 +11,7 @@
     'Definition',
     'Config',
     'resources',
+    'log'
```

Comment:
Do we really want to include `log` into `__all__` ?

Reply:
When i tried to import log module Pycharm could not find log module without it

Reply:
Hm, could you show how you are importing this module?

Reply:
```
import bigflow as bf
bf.log
```

Reply:
Well, 'bigflow.log' is a module, so it should be imported explicitly (with `import bigflow.log` or `from bigflow import log`).  

It might be not a good practice to proactively import all submodules in '__init__':
 - a lot of boilerplate code
 - might introduce some strange bugs when submodule name clashed with some variable inside __init__
 - stop us from using lazy-loading (which may be very convenient for `bigflow`)

Reply:
Done